### PR TITLE
LPS-135177 Make select object read only

### PR DESCRIPTION
**Pull request by mateussgn**
Related issue: https://issues.liferay.com/browse/LPS-135177

### CODE REVIEW

**Comment by mateussgn** [comment]
ci:test:sf

**Comment by mateussgn** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormAdminDisplayContext.java [hunk 1]
```diff
@@ -1240,6 +1240,10 @@ protected DDMForm createSettingsDDMForm(
 			DDMFormField ddmFormField = ddmFormFieldsMap.get("storageType");
 
 			ddmFormField.setReadOnly(true);
+
+			ddmFormField = ddmFormFieldsMap.get("objectDefinitionId");
```
Comment (gabrielwas) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormAdminDisplayContext.java:1244]: @mateussgn can you sort and put objectDefinitionId first? Thanks 😄

Comment (mateussgn) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormAdminDisplayContext.java:1244]: Sure 😃

**Comment by mateussgn** [comment]
ci:stop

**Comment by mateussgn** [comment]
ci:test:relevant

**Comment by gabrielwas** [comment]
LGTM 😄

**Comment by gabrielwas** [comment]
ci:test:relevant

**Comment by gabrielwas** [comment]
ci:test:relevant

**Pull request closed without merging**